docs/spec/lsm/adr-002.md

```diff
@@ -0,0 +1,362 @@
+
+Next, let’s discuss how the LSM makes the adoption of liquid staking more efficient, and can help the blockchain that installs it build strong relationships with liquid staking providers. The LSM enables users to instantly liquid stake their staked tokens, without having to wait the twenty-one day unbonding period. This is important, because a very large portion of the token supply on most Cosmos blockchains is currently staked. Liquid staking tokens that are already staked incur a switching cost in the form of three weeks’ forfeited staking rewards. The LSM eliminates this switching cost.
+
+
+A user would be able to visit any liquid staking provider that has integrated with the LSM and click a button to convert his staked tokens to liquid staked tokens. It would be as easy as liquid staking unstaked tokens.
+
+Technically speaking, this is accomplished by using something called an “LSM share.” Using the liquid staking module, a user can tokenize their staked tokens and turn it into LSM shares. LSM shares can be redeemed for underlying staked tokens and are transferable. After staked tokens are tokenized they can be immediately transferred to a liquid staking provider in exchange for liquid staking tokens - without having to wait for the unbonding period.
+
+
+## Economics:
+
+We expect that eventually governance may decide that the principal agent problems between validators and liquid staking are resolved through the existence of mature liquid staking synthetic asset systems and their associate risk framework. Governance can effectively disable the feature by setting the scalar value to -1 and allow unlimited minting and all liquid delegations to be freely undelegated.
+
+During the transitionary period, this creates a market for liquid shares that may serve to help further decentralize the validator set.
+
+It also allows multiple participants in a validator business to hold their personal stakes in segregated accounts but all collectively contribute towards demonstrating alignment with the safety of the protocol.
+
+
+## Technical Spec
+
+### Modification from ADR-001 
+ADR-002 introduces the global liquid staking cap, validator liquid staking cap, and renames exemption factor to validator bond factor. Delegations from 32-length addresses and LSM tokenized shares are tracked against the global liquid staking, validator liquid staking cap, and validator bond caps. This requires changing the standard staking transactions to introspect. The reason for checking the account type is because ICAs and tokenize share record module accounts have 32-length addresses, so in practice this limits liquid staking. To be clear, any ICA or module account staking is counted against this cap - not just ICA delegations from liquid staking providers.
+
+### Instructions for validators
+Once delegated to a validator, a delegator or validator can convert their delegation to a validator into Validator Bond by signing a ValidatorBond message. 
```

Comment:
> a delegator or validator can convert their delegation

Why a validator? From the code, only a delegator can do that.

Reply:
IIUC this comment refers to the fact that a validator operator can validator bond to their own validator